front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -39,20 +41,14 @@ export default function SpeedLimitByTagSelector({
       >
         <img width="32px" src={icon} alt="speedometer" />
         <span className="text-muted">{t('speedLimitByTag')}</span>
-        <SelectImprovedSNCF
-          sm
+        <Select
           disabled={disabled}
-          withSearch
-          dataTestId="speed-limit-by-tag-selector"
-          value={speedLimitByTag || t('noSpeedLimitByTag').toString()}
-          options={speedLimitsTagsList}
+          id="speed-limit-by-tag-selector"
+          value={speedLimitByTag || ''}
           onChange={(e) => {
-            if (e && e !== t('noSpeedLimitByTag')) {
-              dispatchUpdateSpeedLimitByTag(e);
-            } else {
-              dispatchUpdateSpeedLimitByTag(null);
-            }
+            dispatchUpdateSpeedLimitByTag(e || null);
           }}
+          {...createStringSelectOptions(speedLimitsTagsList, t('noSpeedLimitByTag').toString())}
```

Comment:
Don't think you need to do that, the default one is already added in the list in `useStoreDataForSpeedLimitByTagSelector`

```suggestion
          {...createStringSelectOptions(speedLimitsTagsList)}
```

Reply:
Done